### ISSUE TITLE
Outline openwashdata/book as Quarto book 

### ISSUE THREAD

**Issue by larnsce** [openwashdata/book#4]
# Description of task

The purpose of this book is to document our knowledge as we work through the Work Packages. Each Work Package has a set of activities and research questions that need to be answered. These can be answered within the book and then feed into the progress reporting, blog post, or eventually a scientific publication.

**Comment by mbannert** [comment]
Fine for me. Only concern: external request and our internal stuff might mingle too much.

**Comment by larnsce** [comment]
Yes, you are right. That could be a problem. I have added a tag and thought the title could reflect it, too with [data] at the beginning.

But, how about:

- openwashdata/book -> our internal issue tracker and project management board
- openwashdata/help or openwashdata/data -> the place to submit ideas for data and/or other issues

**Comment by larnsce** [comment]
Setup openwashdata/data now and moved current data issues:

https://github.com/openwashdata/data

Website opens the issue tracker through Help navigation:


![Screenshot 2023-03-08 at 13 59 44](https://user-images.githubusercontent.com/18212549/223719643-837182f4-faa5-44dc-a58e-2f37348a373b.png)
